packages/slate-plugins/src/elements/link/withLink.ts

```diff
@@ -1,6 +1,10 @@
+import { get } from 'lodash';
```

Comment:
For tree-shaking, can you import from `lodash/get`?

Reply:
done